src/Base/Hierarchy.cpp

```diff
@@ -14,26 +14,30 @@ Hierarchy hierarchy;
 void Hierarchy::open_casc(fs::path directory) {
 	warcraft_directory = directory;
 	QSettings settings;
+	ptr = settings.value("flavour", "Retail").toString().toStdString() != "Retail";
```

Comment:
Conversion to toStdString() is not required here

Reply:
Fixed in next commit, thanks for alerting me to this unneeded call